tests/integration-tests/README.md

```diff
@@ -976,3 +976,58 @@ Scheduling:
         SubnetIds:
           - {{ public_subnet_id }}
 ```
+## Collect system analysis
+
+In case of performance degradation detected in tests, to speed up the root cause analysis it is useful to have a
+report of all the services, cron settings and information about the tested cluster. The function `run_system_analyzer` creates a subfolder
+in `out_dir` called `system_analyzer`. Every run the function creates 2 files, one for the head node and one
+for a compute node of the fleet. The file generated is a `tar.gz` archive containing a directory structure
+which is comparable with `diff`. Moreover, the function also collects some network statistics which can be inspected to
+get information about dropped packages and other meaningful network metrics.
+
+The information collected are:
+* System id: ubuntu, amzn or centos
+* System version
+* uname with kernel version
+* Installed packages
+* Services and timers active on the system
+* Scheduled commands like cron, at, anacron
+* The available MPI modules
+* The configured network and the statistics associated to those network
+* ami-id, instance-type and user data queried from the instance metadata service (IMDSv2)
+
+### How to add system analysis to a test
+
+In order to add the system analysis to a test do the following:
+1. Import the function from utils.py (e.g. `from tests.common.utils import run_system_analyzer`)
+2. Call the function after the cluster creation; can be useful to run it after all the other tests.
```

Comment:
`after all the other tests.` --> as the last step of the test.

Reply:
Done